views/templates/front/altcoin.tpl

```diff
@@ -0,0 +1,148 @@
+{*
+ * 2011-2016 Blockonomics
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@blockonomics.co so we can send you a copy immediately.
+ *
+ * @author    Blockonomics Admin <admin@blockonomics.co>
+ * @copyright 2011-2016 Blockonomics
+ * @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+ * International Registered Trademark & Property of Blockonomics
+ *}
+{extends $layout}
+
+{block name="content"}
+
+<div class="form" ng-app="blockonomics-invoice">
+  <script>var ajax_url = "{$ajax_url|escape:'htmlall':'UTF-8'}";</script>
+  <script>var get_uuid = "{$uuid|escape:'htmlall':'UTF-8'}";</script>
```

Comment:
Is this used ? Please remove if unused

Reply:
Removed unused variables. 77145834a021608a34eb96d11ef4658140178d1e